packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -1795,7 +1795,7 @@ export class LinuxBuildImage implements IBuildImage {
    * @deprecated Use LinuxArmBuildImage.AMAZON_LINUX_2_STANDARD_3_0 instead.
```

Comment:
v2 seems not to be deprecated as well.
Can you please update the comment accordingly?